uaclient/jobs/tests/test_update_contract_info.py

```diff
@@ -0,0 +1,59 @@
+import mock
+import pytest
+
+from uaclient.files.notices import Notice
+from uaclient.jobs.update_contract_info import update_contract_info
+
+M_PATH = "uaclient.jobs.update_contract_info."
+
+
+@mock.patch(M_PATH + "contract.is_contract_changed", return_value=False)
+class TestUpdateContractInfo:
```

Comment:
As a suggestion, we could have a test where some error happens, and we check it is logged.

Reply:
done